openam-ui/openam-ui-ria/src/main/resources/locales/ja/translation.json

```
@@ -0,0 +1,1292 @@
+            "search" : "検索",
+            "submit" : "サブミット",
+            "actions" : "アクション",
+            "complete" : "完了",
+            "revoke" : "取り消し",
+            "done" : "終了",
+            "ok" : "OK",
+            "moveUp" : "上に移動",
+            "moveDown" : "下に移動",
+            "standard": "標準",
+            "advanced": "詳細設定",
+            "revert": "元に戻す",
+            "saveChanges": "変更の保存",
+            "edit": "編集",
+            "key":"キー",
+            "value": "値",
+            "addKey" : "キーの追加",
+            "addValue" : "値の追加",
+            "settings" : "設定",
+            "working" : "処理中...",
+            "more" : "more...",
+            "empty" : "空",
+            "help" : "ヘルプ",
+            "change": "変更",
+            "upload": "アップロード",
+            "download": "ダウンロード",
+            "validate": "検証",
+            "filter": "フィルタ...",
+            "pass" : "成功",
+            "fail" : "失敗",
+            "success" : "成功",
+            "allow": "許可",
+            "deny": "拒否",
+            "notApplicable": "n/a",
+            "enable": "有効",
+            "disable": "無効",
+            "enabled": "有効",
+            "disabled": "無効",
+            "activate": "アクティブ",
+            "deactivate": "非アクティブ",
+            "active": "アクティブ",
+            "inactive": "非アクティブ",
+            "copyright" : "Copyright &copy; 2010-2016 ForgeRock AS. All rights reserved.",
+            "build": "ビルド",
+            "sessionExpired" : "セッションが有効期限切れ",
+            "enableCookies" : "ログインするためには、cookie を有効にする必要があります",
+            "deleteSelected" : "選択したものを削除します",
+            "startTypingToSearch": "検索する入力を開始",
+            "passwordPlaceholder": "••••••••••••",
+            "redirecting": "リダイレクトします...",
+            "propertyName": "プロパティ名",
+            "propertyValue": "プロパティ値",
+            "addName": "名の追加",
+            "addValue": "値の追加",
+            "validation" : {
+                "errorsNotSaved" : "フォームに検証エラーが存在します。変更は保存できません。",
+                "VALID_PHONE_FORMAT": "無効な文字が含まれています。",
+                "VALID_NAME_FORMAT": "無効な文字が含まれています。",
+                "VALID_EMAIL_ADDRESS_FORMAT": "電子メールアドレスが無効です。",
+                "AT_LEAST_X_CAPITAL_LETTERS" : "最低 __numCaps__ capital 文字以上は必要です。",
+                "AT_LEAST_X_NUMBERS" : "少なくとも __numNums__ 個以上の数字が必要です。",
+                "CANNOT_CONTAIN_OTHERS" : "値を含めることはできません: __disallowedFields__",
+                "MIN_LENGTH" : "最低 __minLength__ 文字です。",
+                "REQUIRED" : "空白にすることはできません。",
+                "UNIQUE" : "すでに存在します。",
+                "REAUTH_REQUIRED" : "",
+                "formContainsErrors" : "フォームには検証エラーが含まれています。",
+                "atLeastOneCapitalLetter" : "少なくとも大文字が 1 文字以上必要です。",
+                "atLeastOneNumber" : "少なくとも数字が 1 つ以上必要です。",
+                "atLeast8Characters" : "少なくとも 8 文字以上必要です。",
+                "cannotMatchLogin" : "ログインが一致しません。",
+                "confirmationMatchesPassword" : "新規パスワードとパスワード(確認)が一致。",
```

`～一致する必要があります。`

---

I think that it is a reasonable proposal, thank you.
```suggestion
                "confirmationMatchesPassword" : "新規パスワードとパスワード(確認)が一致する必要があります。",
```